code/modules/clothing/suits/toggles.dm

```diff
@@ -0,0 +1,64 @@
+/obj/item/clothing/suit/wintercoat
+    name = "winter coat"
+    desc = "A warm, fluffy and incredibly comfortable jacket made from animal furs."
+    icon_state = "coatwinter"
+    var item_state_slots = list(slot_r_hand_str = "coatwinter", slot_l_hand_str = "coatwinter")
```

Comment:
@Gef21 what he means is you should remove this and replace it with the actual item states and in hand states